src/Annotations/Command/QueuePushCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace eLife\Annotations\Command;
+
+use eLife\Bus\Queue\InternalSqsMessage;
+use eLife\Bus\Queue\WatchableQueue;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class QueuePushCommand extends Command
```

Comment:
also this a candidate for `bus-sdk-php`, but later

Reply:
Yes